### DISCUSSION TITLE
Camel case pagination properties in JSON

### DISCUSSION THREAD

**Discussion by joelvh** [spatie/laravel-data#370] [Q&A]
Hi!

New to laravel-data and I'm wondering how to make the pagination properties in JSON camel case?

Specifically, the `meta` properties are all snake case and [derived](https://github.com/spatie/laravel-data/blob/main/src/Transformers/DataCollectableTransformer.php#L79) from Laravel paginator classes.

Ideally, we could apply the camel case mapper somewhere.

Thank you for your help.

**Comment by Promter** [comment]
I am also new to using Laravel Data, and according to what I understand as Data Transformer Object (DOT), it would be nice to have something like ->toCamelCase();
so that we can implement fast responses.
Is there something like this that I don't know about?
